tests/llmobs/test_llmobs_ragas_evaluators.py

```diff
@@ -238,3 +240,236 @@ def test_llmobs_with_faithfulness_emits_traces_and_evals_on_exit(mock_writer_log
+                metric_type=RagasAnswerRelevancyEvaluator.METRIC_TYPE,
+                value=mock.ANY,
+                metadata={
+                    "_dd.evaluation_span": {"span_id": mock.ANY, "trace_id": mock.ANY},
+                },
+            )
+        ]
+    )
+
+
+@pytest.mark.vcr_logs
+def test_ragas_answer_relevancy_submits_evaluation_on_span_with_custom_keys(
+    ragas, llmobs, mock_llmobs_submit_evaluation, mock_ragas_answer_relevancy_calculate_similarity
+):
+    """Test that evaluation is submitted for a valid llm span where the last message content is the question"""
+    rf_evaluator = RagasAnswerRelevancyEvaluator(llmobs)
+    llm_span = _expected_llmobs_llm_span_event(
+        Span("dummy"),
+        prompt={
+            "variables": {
+                "user_input": "Is france part of europe?",
+                "context_2": "irrelevant",
+                "context_3": "France is part of europe",
+            },
+            "_dd_context_variable_keys": ["context_2", "context_3"],
+            "_dd_query_variable_keys": ["user_input"],
+        },
+        output_messages=[{"content": "France is indeed part of europe"}],
+    )
+    rf_evaluator.run_and_submit_evaluation(llm_span)
+    rf_evaluator.llmobs_service.submit_evaluation.assert_has_calls(
+        [
+            mock.call(
+                span_context={
+                    "span_id": llm_span.get("span_id"),
+                    "trace_id": llm_span.get("trace_id"),
+                },
+                label=RagasAnswerRelevancyEvaluator.LABEL,
+                metric_type=RagasAnswerRelevancyEvaluator.METRIC_TYPE,
+                value=mock.ANY,
+                metadata={
+                    "_dd.evaluation_span": {"span_id": mock.ANY, "trace_id": mock.ANY},
+                },
+            )
+        ]
+    )
+
+
+@pytest.mark.vcr_logs
+def test_ragas_answer_relevancy_emits_traces(
+    ragas, llmobs, llmobs_events, mock_ragas_answer_relevancy_calculate_similarity
+):
+    rar_evaluator = RagasAnswerRelevancyEvaluator(llmobs)
+    rar_evaluator.evaluate(_llm_span_with_expected_ragas_inputs_in_prompt())
+
+    ragas_spans = [event for event in llmobs_events if event["name"].startswith("dd-ragas.")]
+    ragas_spans = sorted(ragas_spans, key=lambda d: d["start_ns"])
+
+    assert len(ragas_spans) == 3
+    # check name, io, span kinds match
+    assert ragas_spans == _expected_ragas_answer_relevancy_spans()
+
+    # verify the trace structure
+    root_span = ragas_spans[0]
+    root_span_id = root_span["span_id"]
+    assert root_span["parent_id"] == "undefined"
+    assert root_span["meta"] is not None
+
+    root_span_trace_id = root_span["trace_id"]
+    for child_span in ragas_spans[1:]:
+        assert child_span["trace_id"] == root_span_trace_id
+        assert child_span["parent_id"] == root_span_id
+
+
+def test_llmobs_with_answer_relevancy_emits_traces_and_evals_on_exit(mock_writer_logs, run_python_code_in_subprocess):
```

Comment:
Feel free to remove this test, I don't really see the value added by these exit tests